rusoto/credential/src/profile.rs

```diff
@@ -18,8 +18,36 @@ const AWS_CONFIG_FILE: &str = "AWS_CONFIG_FILE";
 const AWS_PROFILE: &str = "AWS_PROFILE";
 const AWS_SHARED_CREDENTIALS_FILE: &str = "AWS_SHARED_CREDENTIALS_FILE";
 const DEFAULT: &str = "default";
+/// AWS Region to use when creating clients.
 const REGION: &str = "region";
 
+/// Property name for specifying the Amazon AWS Access Key
```

Comment:
IMHO, not sure about these comments, the variables seems self-explanatory.

Reply:
👍🏽 

Those were the descriptions from aws.